src/styles/common/icons.styl

```diff
@@ -0,0 +1,41 @@
+@font-face
+  font-family: "untitled-font-6"
+  src: url("../../fonts/untitled-font-6.eot")
+  src: url("../../fonts/untitled-font-6.eot?#iefix") format("embedded-opentype"),
+    url("../../fonts/untitled-font-6.woff") format("woff"),
+    url("../../fonts/untitled-font-6.ttf") format("truetype"),
+    url("../../fonts/untitled-font-6.svg#untitled-font-6") format("svg")
+  font-weight: normal
+  font-style: normal
+
+[class^="icon-"]:before,
+[class*=" icon-"]:before
+  font-family: "untitled-font-6"
+  font-style: normal
+  font-weight: normal
+  font-variant: normal
+  text-transform: none
+  line-height: 1
+  -webkit-font-smoothing: antialiased
```

Comment:
What are these for?

Reply:
It was auto generated with icon fonts, but anyway worth to read (http://szafranek.net/blog/2009/02/22/font-smoothing-explained/)